appuselfbot.py

```diff
@@ -408,36 +408,30 @@ def check(msg):
     os._exit(0)
 
 
-@bot.command(pass_context=True, aliases=['upgrade'])
-async def update(ctx, msg: str = None):
-    """Update the bot if there is an update available."""
-    if msg:
-        latest = update_bot(False) if msg == 'show' else update_bot(True)
-    else:
-        latest = update_bot(True)
-    if latest:
-        if not msg == 'show':
-            if embed_perms(ctx.message):
-                try:
-                    await ctx.send(content=None, embed=latest)
-                except:
-                    pass
-            await ctx.send(bot.bot_prefix + 'There is an update available. Downloading update and restarting (check your console to see the progress)...')
+
+@bot.command()
+async def update(self, ctx):
+    '''Auto Update command, checks if you have latest version
+    Use tags github-token to find out how to set up this token'''
+    git = self.bot.get_cog('Git')
+    if not await git.starred('PallavBS/Discord-Selfbot-1'): return await ctx.send('**This command is disabled as the user have not starred <https://github.com/PallavBS/Discord-Selfbot-1>**')

```

Comment:
Holding the update command hostage by requiring that users star your github repo is a very quick way to ensure that this PR does not get through.